CHANGELOG.md

```diff
@@ -2,211 +2,83 @@
 
 ## Unreleased
 
-### Significant change in behavior
-
-- Added `Sentry` namespace to global usings when `ImplicitUsings` is enabled ([#3043](https://github.com/getsentry/sentry-dotnet/pull/3043))
-If you have conflicts, you can opt-out by adding the following to your `csproj`:
-```
-<PropertyGroup>
-  <SentryImplicitUsings>false</SentryImplicitUsings>
-</PropertyGroup>
-```
-
-### Fixes
-
-- Moved the binding to MAUI events for breadcrumb creation from `WillFinishLaunching` to `FinishedLaunching`. This delays the initial instantiation of `app`. ([#3057](https://github.com/getsentry/sentry-dotnet/pull/3057))
-- The SDK no longer adds the `WinUIUnhandledExceptionIntegration` on non Windows platforms ([#3055](https://github.com/getsentry/sentry-dotnet/pull/3055))
-
-### Dependencies
-
-- Bump Java SDK from v7.1.0 to v7.2.0 ([#3049](https://github.com/getsentry/sentry-dotnet/pull/3049))
-  - [changelog](https://github.com/getsentry/sentry-java/blob/main/CHANGELOG.md#720)
-  - [diff](https://github.com/getsentry/sentry-java/compare/7.1.0...7.2.0)
-- Bump CLI from v2.25.0 to v2.25.3 ([#3059](https://github.com/getsentry/sentry-dotnet/pull/3059), [#3062](https://github.com/getsentry/sentry-dotnet/pull/3062))
-  - [changelog](https://github.com/getsentry/sentry-cli/blob/master/CHANGELOG.md#2253)
-  - [diff](https://github.com/getsentry/sentry-cli/compare/2.25.0...2.25.3)
+This major release includes many exciting new features including preview support for [Profiling](https://docs.sentry.io/platforms/dotnet/profiling/) and Metrics(LINK), [AOT](https://sentry.engineering/blog/should-you-could-you-aot) with [Native Crash Reporting](https://github.com/getsentry/sentry-dotnet/issues/2770), [Spotlight](https://spotlightjs.com/), Screenshots on MAUI and much more. Details about these features and other changes below.
```

Comment:
We could do a blog post on the new/experimental metrics support if that would help, in lieu of any product docs for this?

Reply:
Blog is coming soon